src/app/calendar/samplePage.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+
+import React from "react";
+import Calendar, { CalendarEvent as CalendarEvent } from "../../components/Calendar/Calendar";
+import { styled } from "styled-components";
+
+const Centerer = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`;
+const SizedAspectRatio = styled.div`
+    width: 100vmin;
+    height: 100vmin;
+`;
+
+const SampleCalendar: React.FC<{}> = () => {
+    const sampleEvents: CalendarEvent[] = [
+        {
+            id: "a",
+            title: "event1",
+            start: new Date("2023-07-11"), // default Date constructor assumes UTC by default
+            end: new Date("2023-07-12"),
+            backgroundColor: "red",
+            textColor: "black",
+            allDay: true,
+        },
+        {
+            id: "b",
+            title: "event2",
+            start: new Date("2023-07-12"),
+            end: new Date("2023-07-13"),
+            daysOfWeek: [3],
+            startRecur: "2023-07-12",
```

Comment:
the recur field is only used when there is a "daysofweek" field which allows events to be repeated. For example in this case, the event happens on every Wednesday. But the first occurrence of the event is on 12 July, and not any earlier.

Reply:
where is startRecur / endRecur actually used, though? I didn't see anything for it

(no, the names are good enough for there not to be comments!)

Reply:
It's used by full calendar (as an Event prop)

Reply:
do we need recurring events?
I think we will just be reading in times from the database and plotting them on the calendar, rather than scheduling recurring meetings etc

Reply:
That's right. We have removed the recurring fields.